src/ModulePlan.js

```diff
@@ -5,19 +5,52 @@ class ModulePlan extends React.Component {
   render() {
     return (
       <div>
-        <section id="symbology">
-          Legende
-        </section>
+      <section id="symbology">
         <div>
-          Regelstudienplan
-          {this.props.semesters.map(function(semester, index) {
-            return (
-              <Semester semester={index+1} modules={semester} />
-            )
-          })
-          }
-        </div>
+
+        <legend>
+          <div className="flex-container">
+            <div className="flex-item item1">
+              <div className="available"></div>
+              <div><h6>Kurs belegt</h6></div>
```

Comment:
"Kurs belegbar" or "Kurs verfügbar" would be better fitting, please adjust that :)